test_bandwidth.py

```diff
@@ -0,0 +1,204 @@
+import os
+import sys
+import time
+import argparse
+from abc import abstractmethod
+
+import torch
+import torch.distributed as dist
+from typing import List
+from prettytable import PrettyTable
+os.environ['CUDA_VISIBLE_DEVICES'] = '1,2,3,4'
```

Comment:
You should not set this in the script. You should just do it in shell.

Reply:
I see, I'll move it to the shell.